src/vue-scanner.ts

```diff
@@ -777,11 +777,11 @@ export class VueScanner implements Scanner {
 	}
 
 	/**
- 	* Write an array of component profiles to a JSON file within the specified app directory.
+		* Write an array of component profiles to a JSON file within the specified app directory.
```

Comment:
@ALLENGJOSE Thank you for your contribution! To maintain consistent code formatting, could you please adjust the code indentation to use 2 spaces? Your attention to this detail is greatly appreciated.

Reply:
yes sure